README.md

```diff
@@ -9,7 +9,6 @@ applications. Some of the features demonstrated in this example are:
 - Custom [devicetree bindings][bindings]
 - Out-of-tree [drivers][drivers]
 - Out-of-tree libraries
-- Documentation using Doxygen and Sphinx
```

Comment:
... and removed in this one. Rebase error?

Can you please squash this hunk into the previous commit?

Reply:
Done.